output/cloud/expv2/mapping.go

```diff
@@ -10,21 +11,41 @@ import (
 )
 
 // TODO: unit test
-func mapTimeSeriesLabelsProto(tags *metrics.TagSet) []*pbcloud.Label {
+func mapTimeSeriesLabelsProto(tags *metrics.TagSet) ([]*pbcloud.Label, []string) {
 	labels := make([]*pbcloud.Label, 0, ((*atlas.Node)(tags)).Len())
+	var discardedLabels []string
 
-	// TODO: move it as a shared func
+	// TODO: move this as a shared func
 	// https://github.com/grafana/k6/issues/2764
 	n := (*atlas.Node)(tags)
 	if n.Len() < 1 {
-		return labels
+		return labels, discardedLabels
 	}
 	for !n.IsRoot() {
 		prev, key, value := n.Data()
-		labels = append(labels, &pbcloud.Label{Name: key, Value: value})
+
 		n = prev
+		if !isReservedLabelName(key) {
+			labels = append(labels, &pbcloud.Label{Name: key, Value: value})
+			continue
+		}
+
+		if discardedLabels == nil {
+			discardedLabels = make([]string, 0, 1)
+		}
+
+		discardedLabels = append(discardedLabels, key)
 	}
-	return labels
+	return labels, discardedLabels
+}
+
+func isReservedLabelName(name string) bool {
+	// this is a reserved label prefix for the prometheus
```

Comment:
```suggestion
	// this is a reserved label pattern for Prometheus, that is used in the Cloud's remote service.
```
Someone of us could ask.."ok, why do we care about it in k6 which doesn't use Prometheus?"